subsys/bluetooth/controller/ll_sw/ull_conn_internal.h

```diff
@@ -17,6 +17,9 @@
  */
 #define PAYLOAD_OVERHEAD_SIZE (2 + 4)
 
+#define PHY_1M (BIT(0))
```

Comment:
Drop the paranthesis around BIT(x). 

Otherwise the PR LGTM.

Reply:
Parentheses removed